src/pyodide/python-entrypoint-helper.ts

```diff
@@ -164,26 +165,90 @@ function makeHandler(pyHandlerName: string): Handler {
     }
   };
 }
+
+function makeDurableObjectClass(className: string, classKind: AnyClass) {
+  class DurableObjectWrapper extends classKind {
+    pyInstance: Promise<PyModule>;
+
+    constructor(...args: any[]) {
+      super(...args);
+      // Initialise a Python instance of the class.
+      this.pyInstance = this.initPyInstance(args);
+      // We do not know the methods that are defined on the RPC class, so we need a proxy to
+      // support any possible method name.
+      return new Proxy(this, {
+        get(target, prop, receiver) {
```

Comment:
Not clear to me that this respects [the invariants required of a get handler](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/Proxy/get#invariants). But we can probably fix any problems later as they come up.